### PR TITLE
Add divide 255 pre processor test

### PR DESCRIPTION
**Pull request by Joeper214**
## What this patch does to fix the issue.
Increase code coverage by adding pre_processor unit test code for Divide255 Class.

### CODE REVIEW

File: tests/unit/test_pre_processor.py [hunk 1]
```diff
@@ -45,6 +46,20 @@ def test_resize():
...
+def test_divide_255():
+    orig_image = np.full((1, 1, 3), 1)
+    expect = np.array([[[0.00392157, 0.00392157, 0.00392157]]])
+
+    pre_processor = DivideBy255()
+    processed = pre_processor(image=orig_image)
+    processed_imaged = processed["image"]
+
+    assert isinstance(processed_imaged, np.ndarray)
+    assert isinstance(processed_imaged, np.ndarray)
```
Comment (tfujiwar) [inline tests/unit/test_pre_processor.py]: Remove one of them.

Comment (Joeper214) [reply tests/unit/test_pre_processor.py]: Fixed! thanks!

File: tests/unit/test_pre_processor.py [hunk 2]
```diff
@@ -45,6 +46,19 @@ def test_resize():
     assert resized_mask.shape[2] == 3
 
 
+def test_divide_255():
+    orig_image = np.full((1, 1, 3), 1)
```
Comment (iizukak) [inline tests/unit/test_pre_processor.py]: To initialize array with 1, I prefer to use `np.ones`

Comment (iizukak) [inline tests/unit/test_pre_processor.py]:
```suggestion
    orig_image = np.ones((1, 1, 3))
```

Comment (Joeper214) [reply tests/unit/test_pre_processor.py]: fixed. Thanks!

**Pull request merged**